tox.ini

```diff
@@ -18,6 +18,7 @@ conda_deps =
     mysqlclient
     ; these packages don't install cleanly with pip, conda has patches
     python-ligo-lw
+    blas=*=openblas
```

Comment:
I think the blas  statements needs clarification since I presume it requires some additional explanation. I'd also ask that (if it has not already been done) an issue be raised on the package causing the underlying issue here (if it is known). It would be good to keep the record of when this line should be removed.

BTW,  what is the timescale for getting those patches into python-ligo-lw for it to build properly? I don't think it's an acceptable solution to rely on conda packages with extra patches. I can understand intermediate periods where things may get fixed in one place and not yet another though. Is that the situation here?

Reply:
Hah, I see @spxiwh you asked the same questions essentially afterwards. That's what I get for leaving the browser window open before teaching class and only answering afterwards.

Reply:
@ahnitz if you are referring to the presence of `python-ligo-lw` in `tox.ini`, that is going to be addressed by #5019.